client/src/index.tsx

```diff
@@ -16,7 +16,7 @@ root.render(
   <ThemeProvider theme={theme}>
     {/* CssBaseline kickstart an elegant, consistent, and simple baseline to build upon. */}
     <CssBaseline />
-    <App />
```

Comment:
Interesting hahahaha, maybe the distant future we can simply bounce them out to a different page. But creative way of doing it LOL